app/controllers/works_controller.rb

```diff
@@ -785,15 +785,23 @@ def update_multiple
     end
   end
 
-  # marks a work to read later, or unmarks it if the work is already marked
+  # marks a work to read later
   def marktoread
     @work = Work.find(params[:id])
-    Reading.mark_to_read_later(@work, current_user)
-    read_later_path = user_readings_path(current_user, :show => 'to-read')
+    Reading.mark_to_read_later(@work, current_user, true)
+    read_later_path = user_readings_path(current_user, show: 'to-read')
     if @work.marked_for_later?(current_user)
-      flash[:notice] = ts("This work was <strong>removed</strong> from your #{view_context.link_to('Marked for Later list', read_later_path)}. It may take a while for changes to show up.").html_safe
-    else
-      flash[:notice] = ts("This work was <strong>added</strong> to your #{view_context.link_to('Marked for Later list', read_later_path)}. It may take a while for changes to show up.").html_safe
+      flash[:notice] = ts("This work was <strong>added</strong> to your #{view_context.link_to('Marked for Later list', read_later_path)}.").html_safe
```

Comment:
I think we can remove the `<strong>` tags for both messages now that we're fixing the behavior that necessitated them.